app/src/main/res/layout/activity_settings.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".view.activities.SettingsActivity">
+
+    <android.support.v7.widget.AppCompatTextView
+        android:id="@+id/tv_about"
```

Comment:
We're using ids like this `tvAbout` and not like `tv_about`. Can you change that here and everywhere else?